utility/utils.py

```diff
@@ -1273,3 +1273,20 @@ def method_should_succeed(function, *args, **kwargs):
     log.debug(f"The {function} return status is {rc}")
     if not rc:
         raise AssertionError(f"Execution failed at function {function}")
+
+
+def clone_the_repo(config, node, path_to_clone):
+    """
+    clone the repo on to test node
```

Comment:
Since we have hosted CephCI documentation, can we format doc-string as per google style?
Ref: https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google

Reply:
docs added as per google style.